src/lib/components/hamburger-menu/menu.svelte

```diff
@@ -61,7 +61,7 @@
         {#if !username}
           <a
             data-test-id="login-btn"
-            class="default-transition rounded-xl bg-skin-text px-4 py-2 text-skin-off-background hover:bg-primary-100 dark:bg-skin-text-highlight dark:text-skin-background dark:hover:bg-primary-100"
+            class="default-transition mt-5 rounded-xl bg-skin-text px-4 py-2 text-skin-off-background hover:bg-primary-100 dark:bg-skin-text-highlight dark:text-skin-background dark:hover:bg-primary-100"
```

Comment:
I think we should be consistent with margins and use the same number here as above

Reply:
Got it. Will change the following